src/index.jsx

```diff
@@ -7,17 +7,18 @@ import { Provider } from 'react-redux';
 import CurrentPostsPage from './containers/CurrentPostsPage';
 import DisclosurePage from './components/DisclosurePage';
 import store from './data/store';
+import styles from './App.scss';
 
 const App = () => (
   <Provider store={store}>
     <Router>
       <div>
         <header>
           <nav>
-            <ul>
-              <li><Link to="/">Home</Link></li>
-              <li><Link to="/posts">Posts</Link></li>
-              <li><Link to="/disclosure">Disclosure</Link></li>
+            <ul className="nav">
+              <li className="nav-item"><Link className="nav-link" to="/">Home</Link></li>
+              <li className="nav-item"><Link className="nav-link" to="/posts">Posts</Link></li>
+              <li className="nav-item"><Link className="nav-link" to="/disclosure">Disclosure</Link></li>
```

Comment:
studio-frontend uses css modules for specifying bootstrap classes. E.g.:

```jsx
import edxBootstrap from 'edx-boostrap.scss';
...
<ul className=edxBootstrap['nav']>
```

Just noting the difference... I don't think it really gives any benefit to do that since the Bootstrap classes aren't going to change. So I *think* this is better.